server/server.go

```diff
@@ -510,7 +505,10 @@ func New(config *Config) (*Server, error) {
 		return nil, err
 	}
 
-	containerServer := libkpod.New(r, store, imageService, config.SignaturePolicyPath)
+	containerServer, err := libkpod.New(libkpod.DefaultConfig())
```

Comment:
Also, since we're now setting up c/image and c/storage stuff inside libkpod, can probably remove that stuff from server.